rust/core-lib/src/view.rs

```diff
@@ -208,16 +208,11 @@ impl View {
                 self.do_gesture(text, line, col, ty),
             GotoLine { line } => self.goto_line(text, line),
             Find { chars, case_sensitive, regex, whole_words } =>
-                self.do_find(text, chars, case_sensitive, regex.unwrap_or(false),
-                             whole_words.unwrap_or(false)),
+                self.do_find(text, chars, case_sensitive, regex, whole_words),
             FindNext { wrap_around, allow_same, modify_selection } =>
-                self.find_next(text, false, wrap_around.unwrap_or(false),
-                               allow_same.unwrap_or(false),
-                               &modify_selection.unwrap_or(SelectionModifier::Set)),
+                self.find_next(text, false, wrap_around, allow_same, &modify_selection),
```

Comment:
Ah I actually have one more nit while we're in here; by convention, we preface the names of functions that are directly handling RPCs with `do_`, so this should be `do_find_next`, and below should be `do_selection_for_find`, etc.

Reply:
Done